org-formation/730-cloudwatch-to-S3/_tasks.yaml

```diff
@@ -0,0 +1,40 @@
+Parameters:
+  <<: !Include '../_parameters.yaml'
+
+  appName:
+    Type: String
+    Default: 'CloudWatch2S3'
+
+  accountId:
+    Type: String
+    Description: The identifier from the account used as the main account
+    Default: !Ref LogCentralAccount
+
+Cloudwatch2S3-bucket:
+  Type: update-stacks
+  Template: CloudWatch2S3-bucket.yaml
+  StackName: !Sub '${resourcePrefix}-${appName}-bucket'
+  DefaultOrganizationBindingRegion: !Ref primaryRegion
+  DefaultOrganizationBinding:
+    Account: !Ref accountId
+
+Cloudwatch2S3-main:
+  Type: update-stacks
+  Template: CloudWatch2S3.yaml
```

Comment:
I don't like the idea of referring directly to upstream templates because we don't control what's in them and also the github repo could just disappear as well.

Reply:
Changed to deploy the Cloudwatch2S3 solution from the AWS serverless repository.  It doesn't work yet due to Cloudwatch2S3 issue https://github.com/CloudSnorkel/CloudWatch2S3/issues/14